GVFS/GVFS.UnitTests/Common/NuGetUpgrade/OrgNuGetUpgraderTests.cs

```diff
@@ -0,0 +1,175 @@
+using GVFS.Common;
+using GVFS.Common.Git;
+using GVFS.Common.NuGetUpgrade;
+using GVFS.Common.Tracing;
+using GVFS.Tests.Should;
+using GVFS.UnitTests.Category;
+using GVFS.UnitTests.Mock.Common;
+using GVFS.UnitTests.Mock.FileSystem;
+using Moq;
+using Moq.Protected;
+using NuGet.Packaging.Core;
+using NuGet.Protocol.Core.Types;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace GVFS.UnitTests.Common.NuGetUpgrade
+{
+    [TestFixture]
+    public class OrgNuGetUpgraderTests
+    {
+        private const string CurrentVersion = "1.5.1185.0";
+        private const string NewerVersion = "1.6.1185.0";
+
+        private const string DefaultUpgradeFeedPackageName = "package";
+        private const string DefaultUpgradeFeedUrl = "https://pkgs.dev.azure.com/contoso/";
+        private const string DefaultOrgInfoServerUrl = "https://www.contoso.com";

```

Comment:
Why do these need to be real URLs in a unit test?

Reply:
`DefaultUpgradeFeedUrl` is parsed and interpreted, so it needs to follow some requirements. Right now, this pattern is hardcoded, but is something we could look at relaxing.

`DefaultOrgInfoServerUrl` could probably be anything.